apps/files/src/views/Sidebar.vue

```diff
@@ -17,14 +17,20 @@
 		@closing="handleClosing"
 		@closed="handleClosed">
 		<template v-if="fileInfo" #subname>
-			<NcIconSvgWrapper v-if="fileInfo.isFavourited"
-				:path="mdiStar"
-				:name="t('files', 'Favorite')"
-				inline />
-			{{ size }}
-			<NcDateTime :timestamp="fileInfo.mtime" />
+			<div class="sidebar__subname">
+				<NcIconSvgWrapper v-if="fileInfo.isFavourited"
+					:path="mdiStar"
+					:name="t('files', 'Favorite')"
+					inline />
+				<span>{{ size }}</span>
+				<span class="sidebar__subname-separator">.</span>
+				<NcDateTime :timestamp="fileInfo.mtime" />
+				<span class="sidebar__subname-separator">.</span>
```

Comment:
Should probably use the same separator as versions

```suggestion
				<span class="sidebar__subname-separator">•</span>
				<NcDateTime :timestamp="fileInfo.mtime" />
				<span class="sidebar__subname-separator">•</span>
```